src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -31,9 +36,34 @@ export class CourseOverviewComponent implements OnInit {
             this.courseService.findAll().subscribe((res: HttpResponse<Course[]>) => {
                 this.courseCalculationService.setCourses(res.body);
                 this.course = this.courseCalculationService.getCourse(this.courseId);
+                this.adjustCourseDescription();
             });
         } else {
             this.course = this.courseCalculationService.getCourse(this.courseId);
         }
+        this.adjustCourseDescription();
+    }
+
+    adjustCourseDescription() {
+        debugger;
```

Comment:
I don't think we want this in the codebase ;)

Reply:
done